distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc

```diff
@@ -16,12 +16,19 @@
 |false
 |true
 
-|Copy Files to Backup Directory
-|copyIngestedFiles
-|Boolean
-|Optional: Copy the ingested files into a backup directory under the monitored directory named /.ingested  -  NOTE: this will double the amount of disk space required for ingested files in this monitored directory if its Processing Directive includes storing the file in the DDF Content Repository.
-|
-|false
+|Maximum Concurrent Files
+|numThreads
+|Integer
+|Specifies the maximum number of concurrent files to be processed within a directory (maximum of 8).  If this number exceeds 8, 8 will be used in order to preserve system resources. Make sure that your system has enough memory to support the number of concurrent processing threads across all directory monitors.
+|1
+|true
+
+|ReadLock Time Interval
+|readLockIntervalMilliseconds
+|Integer
+|Specifies the time to wait (in milliseconds) before acquiring a lock on a file in the monitored directory.  This interval is used for sleeping between attempts to acquire the read lock on a file to be ingested.  The default value of 100 milliseconds is recommended.
+|100
+|true
 
```

Comment:
Should details for [`processingMechanism`](https://github.com/codice/ddf/blob/7c702efa12c64098cefebcc244440a55044c02b6/catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml#L32-L39) be added to this table?